File: Lab2/calculator.js
var array = process.argv[2].split(" ");

var in1 = parseInt(array[0], 10);
var in2 = parseInt(array[2], 10);

if (array[1] == '+')
	console.log(in1 + in2);
else if (array[1] == '-')
	console.log(in1 - in2);
else if (array[1] == '*')
	console.log(in1 * in2);
else if (array[1] == '/')
	console.log(in1 / in2);

File: Lab2/calculator.java
//Basic user input calculator for Java 
public class Calculator {
	public static void main(String[] args) {
		String[] input = args[0].split(" ");
		int in1 = Integer.parseInt(input[0]);
		int in2 = Integer.parseInt(input[2]);
		int out;

		if (input[1].equals("+")) out = in1 + in2;
		else if(input[1].equals("-")) out = in1 - in2;
		else if(input[1].equals("*")) out = in1 * in2;
		else if(input[1].equals("/")) out = in1 / in2;
		else out = in1 / in2;

		System.out.println(args[0] + " = " + out);
	}

	
}